app/views/project_groups/index.haml

```diff
@@ -20,24 +20,43 @@
     %li Access to a project group does not imply access to the projects within the group, projects must still be assigned to users to grant access.
 - if @pagy.count.positive?
   = render 'common/pagination_top', item_name: 'project group'
-  .table-responsive
-    %table.table.table-striped
-      %thead
-        %tr
-          %th Name
-          %th Project Count
-          %th Universe
-      %tbody
-        - @project_groups.each do |group|
+
+  - input_id = "selections[group][]"
+  = simple_form_for :selections, url: delete_multiple_project_groups_path do |f|
+    .table-responsive
+      %table.table.table-striped
+        %thead
           %tr
-            %td
-              = link_to group.name.presence || 'unnamed', edit_project_group_path(group)
-              - if group.used_for_cas_sync?
-                %span.badge.badge-success.ml-3{data: {toggle: :tooltip, title: "Used for syncing clients to CAS"}}
-                  CAS
-            %th= group.projects.count
-            %td.reports.report-parameters-all
-              = group.describe_filter_as_html
+            %th
```

Comment:
@awisema can you add a select all/none option here?  See `init.js.coffee` line 14.  Basically you should be able to add a checkbox with `.jCheckAll` and a unique ID, then add the id as a class to all the checkboxes in the table.

Reply:
Done.